### DISCUSSION TITLE
Show us your tools!

### DISCUSSION THREAD

**Discussion by aderusha** [aderusha/DDD-Printable-Wall-Control-System#12] [Show and tell]
If you've been printing and utilizing parts of this system, take a photo and show us how it turned out!

![Installed DDD Printable Wall Control System](https://i.imgur.com/zUs2Ysu.jpg)

![Installed DDD Printable Wall Control System](https://i.imgur.com/EWCd3iF.jpg)

![Installed DDD Printable Wall Control System](https://i.imgur.com/82tJx73.jpg)

![Installed DDD Printable Wall Control System](https://i.imgur.com/hRrCFp1.jpg)

![Installed DDD Printable Wall Control System](https://i.imgur.com/dReslAw.jpg)

**Comment by cryptohazey** [comment]
This is awesome. Thank you so much for sharing this for free. <3 I really appreciate it dude! Quick question, if you don't mind me asking what are you using for your large clamps and saw tracks?

**Comment by aderusha** [comment]
The tracks are being held by a [cheap bent-wire pegboard hook](https://amzn.to/3BHgp2U) which fits the hole provided in the track.

The parallel clamps are HEAVY and so I built out the following recipe to hold them securely:

* 1x [Sidepieces/Square_brackets/2x4.25 Square Bracket Hook Left.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Square_brackets/2x4.25%20Square%20Bracket%20Hook%20Left.stl)
* 1x [Sidepieces/Square_brackets/2x4.25 Square Bracket Hook Right.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Square_brackets/2x4.25%20Square%20Bracket%20Hook%20Right.stl)
* 1x [Sidepieces/Square_brackets/2x4.25 Square Bracket Hook Center.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Square_brackets/2x4.25%20Square%20Bracket%20Hook%20Center.stl)
* 2x [Centerpieces/Spacer_blank/2x1 Spacer blank.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Spacer_blank/2x1%20Spacer%20blank.stl)
* 4x [Centerpieces/Spacer_blank/4x1 Spacer blank.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Spacer_blank/4x1%20Spacer%20blank.stl)
* 2x [Centerpieces/Locking_spacer/2x1 Locking Spacer.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Locking_spacer/2x1%20Locking%20Spacer.stl)
* 2x [Accessories/8mm Lock Pin.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/8mm%20Lock%20Pin.stl)

Here's what that looks like assembled:
![image](https://user-images.githubusercontent.com/9554705/137161841-71f8b6f6-2192-4d51-8943-5defbfa415c4.png)


And here's an explosion showing the individual parts:
![image](https://user-images.githubusercontent.com/9554705/137161828-e887606d-7fa3-444e-b538-ab6a68220913.png)

**Reply by 1DontEx1st** [reply]
I have the white WC boards and love the look of your hooks - which brand(s) of white/blue material are you using?

**Reply by aderusha** [reply]
> I have the white WC boards and love the look of your hooks - which brand(s) of white/blue material are you using?

I've been having good luck with Overture PLA Pro from Amazon, reasonably priced, ships fast, and reasonably consistent.  I'm using [Digital Blue](https://amzn.to/3AC3cch) and [White](https://amzn.to/3FSdA0f) in the prints above.

**Reply by rgeyer** [reply]
So.. How do you thread in the locking pins to those 1x2 locking center pieces?

I've been assembling my recipe on the bench, then putting it on the board, otherwise things fit too tightly together (a good thing).

In this case, the non locking 1x2 at the front would block access to it.

Maybe replace the front with a 1x2 perforated? Are those holes big enough to pass the locking pin through?

**Reply by aderusha** [reply]
Ugh, yeah that was a mistake on my part.  On my setup I'm using the locking 1x2 in the front and the back, then running the lock pin through the front all the way through, then to the back.  Helps to have a quarter inch square drive on an extension :D

**Comment by DraxStorm** [comment]
I've been playing around with it as well, starting to do more customization for specific tools which I have to upload at some poin
![20211113_171352](https://user-images.githubusercontent.com/78496937/141660662-1fdb4f95-3495-4e17-b538-4deb3a90f831.jpg)
t
![20211113_171402](https://user-images.githubusercontent.com/78496937/141660663-db826845-fdb0-4049-b4a2-48bfa005de0f.jpg)
.

**Reply by aderusha** [reply]
**Oh yeah baby now that's what I like!**  Looking great my man and glad to see Team Yellow represent :D  How do you like that little circ saw?  I've been eyeballing that for a minute, just seems like a handy solution for rip cuts along sheet stock or stuff like that.

I recently added these two models which you might find interesting:

* [4x6 Medium Pliers Rack](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/4x6%20Medium%20Pliers%20Rack.stl)
* [4x4 Large Pliers Rack](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/4x4%20Large%20Pliers%20Rack.stl)

They work a treat for packing in wrenches/pliers vertically, you can fit a lot of tools in a small space and they're all easy to grab.  The only thing this doesn't handle well are tools with sprung handles, as the handles aren't fully constrained in order to make it print without supports.

Here's what that looks like:

![Pliers racks](https://i.imgur.com/AxdQVFg.jpg)

**Reply by DraxStorm** [reply]
LoL, I actually have not used the little circ saw yet.  It was a good price and my father used to have the Makita one which I liked so I'm hoping it will be as useful.  I'm thinking when I need to do thinks like trim back flooring to straighten it when I put flooring in the next room, or anything where I'm not in the workshop with my big tools.
I like the plier idea spacewise, but for now I have to stick with mine since they are labeled and when I'm working upstairs and send my wife or daughter to fetch a pair of pliers it's easier to say "get me the linemans pliers" since there is a label on it, than "get me the pliers that look look big and bulky and have a cutter in them and and are kinds square at the end" ;)  I uploaded the ones I made as well.  I just use some raised text then do a sot of grave rubbing across it with a permeant pen so it's readable. But I do think it may work great for the channel lock and vice grip pliers though, want to get rid of the metal hooks, especially since they like to hold on to the tools and end up on the floor every time I take one off.
You definitely have a beautiful setup there.  I'm jealous, it looks like you have actual space and even windows for natural light.  Mine is the furnace room so I am already running out of space and it's dark and a little cramped.

**Reply by 1DontEx1st** [reply]
My favorite part - the Lowe's kid's kits you have hanging :)

**Reply by pudljmpr** [reply]
As Im new to this system can you help me out and tell me what you are using with the plier racks to attach them to the WC panels. So I guess what is the recipe. Thanks

**Reply by aderusha** [reply]
Hey @pudljmpr !  In the photo above you'll see two "medium pliers rack" that holds 6 hand tools and one "large pliers rack" that holds 4.

Here's a recipe for the medium plier rack that could get you started:
* Centerpiece: [4x6 Medium Pliers Rack.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/4x6%20Medium%20Pliers%20Rack.stl)
* Side piece: [4x0 Flat Left.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/4x0%20Flat%20Left.stl)
* Side piece: [4x0 Flat Right.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/4x0%20Flat%20Right.stl)

For the larger model:
* Centerpiece: [4x4 Large Pliers Rack.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Centerpieces/Tool_hooks/4x4%20Large%20Pliers%20Rack.stl)
* Side piece: [4x0 Flat Left.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/4x0%20Flat%20Left.stl)
* Side piece: [4x0 Flat Right.stl](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/4x0%20Flat%20Right.stl)

If you want to mount these side-by-side, you can use the recipe above as-is and it will work fine, or you can stick the racks together using a "center" side-piece [like this](https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/4x0%20Flat%20Center.stl).

**Comment by rgeyer** [comment]
I've been lurking for a while, and my board is still a work-in-progress. I wanted to share, and express my gratitude again to @aderusha for designing and making this available. I've built a few of my own ad-hoc solutions for the wall control boards, but the modularity of this is perfect!

I have designed a handful of new centerpieces for my tools, with more on the way. In the screenshot below, from left to right are.

1. Hammer head catch (neither of the existing sizes worked for me, and I had to redesign this one thrice to get a good fit)
2. Amazon Echo Dot mount. I haven't actually printed the latest revision, but I kept getting the radius of the edges wrong 🤦‍♂️ 
3. Milessey laser measure.
4. Sharpie rack for 3 king size, and two "regular" fine point markers. I printed the one that's included in the repo, only to realize that I didn't have Magnum markers. This fit my needs better holding two different sizes.

![image](https://user-images.githubusercontent.com/428415/214143085-54d60b83-bc12-46be-835c-9a1060487b1d.png)

Anyway, without further delay, here's my current tool setup. I'll likely post again when I get it "finished", but for now I'm still printing parts.

![PXL_20230123_012255461](https://user-images.githubusercontent.com/428415/214143376-002f8340-b84a-4875-ac3f-bef9ca139ecb.jpg)
